tests/test_account.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+
+from account import BankAccount
```

Comment:
After making the `app` and `tests` folders packages, You should access this class by importing it from the app package i.e
```py
from app.account import BankAccount
```